Documentation/DocuBlocks/Rest/Graph/general_graph_vertex_collection_add_http_examples.md

```diff
@@ -1,30 +1,101 @@
 @startDocuBlock general_graph_vertex_collection_add_http_examples
 @brief Add an additional vertex collection to the graph.
 
-@RESTHEADER{POST /_api/gharial/{graph-name}/vertex, Add vertex collection}
+@RESTHEADER{POST /_api/gharial/{graph}/vertex, Add vertex collection}
 
 @RESTDESCRIPTION
-Adds a vertex collection to the set of collections of the graph. If
-the collection does not exist, it will be created.
+Adds a vertex collection to the set of orphan collections of the graph.
+If the collection does not exist, it will be created.
 
 @RESTURLPARAMETERS
 
-@RESTURLPARAM{graph-name,string,required}
+@RESTURLPARAM{graph,string,required}
 The name of the graph.
 
 @RESTRETURNCODES
 
 @RESTRETURNCODE{201}
-Returned if the edge collection could be added successfully and
-waitForSync is true.
+Is returned if the collection could be created and waitForSync is enabled
+for the `_graphs` collection, or given in the request.
+The response body contains the graph configuration that has been stored.
+
+@RESTREPLYBODY{error,boolean,required,}
+Flag if there was an error (true) or not (false)
+It is false in this response.
+
+@RESTREPLYBODY{code,integer,required,}
+The response code.
+
+@RESTREPLYBODY{graph,object,required,graph_representation}
+The information about the modified graph.
 
 @RESTRETURNCODE{202}
-Returned if the edge collection could be added successfully and
-waitForSync is false.
+Is returned if the collection could be created and waitForSync is disabled
+for the `_graphs` collection, or given in the request.
+The response body contains the graph configuration that has been stored.
+
+@RESTREPLYBODY{error,boolean,required,}
+Flag if there was an error (true) or not (false)
+It is false in this response.
```

Comment:
Should this be here? I see this multiple times. If it is correct, then we should add a dot (.) after (false).